src/surfaces/filtersRail/filtersRailRow.jsx

```diff
@@ -0,0 +1,178 @@
+import {
+    map,
+} from 'lodash';
+import ClassNames from 'classnames';
+import React, { useState } from 'react';
+import {
+    PROP_TYPES_ROW,
+} from './constants';
+import {
+    BEM_FILTERS_RAIL_ROW, BEM_FILTERS_RAIL_ROW_COMONENT,
```

Comment:
This is kinda strange ... these should be on separate lines, right?

Reply:
Yes.

Reply:
Fixed.

Reply:
Looks good.  Resolving.